python/ray/serve/_private/client.py

```diff
@@ -379,11 +379,6 @@ def get_deployment_info(
             deployment_route.route if deployment_route.route != "" else None,
         )
 
-    @_ensure_connected
-    def get_app_config(self, name: str = SERVE_DEFAULT_APP_NAME) -> Dict:
-        """Returns the most recently requested Serve config."""
-        return ray.get(self._controller.get_app_config.remote(name))
```

Comment:
This seems to be unused, so I went ahead and removed it instead of refactoring it. But I'm happy to provide an equivalent method in this PR if desired.